text/0089-metrics-views-api.md

```diff
@@ -0,0 +1,264 @@
+
+Said differently: given an aggregation function `agg` and sequence of measurements `S`, there should exist a function `merge` such that:
+
+```
+agg(S) = merge(agg(S_1), agg(S_2), ..., agg(S_N))
+```
+
+For every partition `{S_1, ..., S_N}` of `S`.
+
+For example, the _min_ aggregation is trivially mergeable:
+
+```
+min([1, 2, 3, 4, 5, 6]) = min([min([1]), min([2, 3]), min([4, 5, 6])])
+```
+
+but quantile aggregations, e.g. _p95_ and _p99_ are not.
+Applications that export quantile metrics should use a mergeable aggregations such as [DDSketch](https://arxiv.org/abs/1908.10693), which estimates quantile values with bounded errors, or export raw measurements without aggregation and compute exact quantiles on the backend.
+
+We require aggregations to be mergeable so that they produce the same results regardless of the collection interval, or the number of collection events per export interval.
+
+### Aggregating across label keys
+
+Every measurement is associated with a _LabelSet_, a set of key-value pairs that describes the environment in which the measurement was captured.
+Label keys and values may be extracted from the [correlation context](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-correlationcontext.md) at the time of measurement.
+They may also be set by the user [at the time of measurement](https://github.com/open-telemetry/opentelemetry-specification/blob/f4aeb318a5b77c9c39132a8cbc5d995e222d124f/specification/api-metrics-user.md#direct-instrument-calling-convention) or at the time at which [the metric instrument is bound](https://github.com/open-telemetry/opentelemetry-specification/blob/f4aeb318a5b77c9c39132a8cbc5d995e222d124f/specification/api-metrics-user.md#bound-instrument-calling-convention).
+
+For each registered view, in each collection interval, measurements with the same labelset are aggregated together.
+
+The view API should allow the user to specify the set of label keys to track for a given view.
+Other label keys should be dropped before aggregation.
+By default, if the user doesn't specify a set of label keys to track, the aggregator should record all label keys.
+
+Users have three options to configure a view's label keys:
+
+1. Record all label keys.
+    This is the default option, and the behavior of the "ungrouped" integrator in [opentelemetry-specification#347](https://github.com/open-telemetry/opentelemetry-specification/pull/347).
+2. Specify a set of label keys to track at view creation time, and drop other keys from the labelsets of recorded measurements before aggregating.
+    This is the behavior of the "defaultkeys" integrator in [opentelemetry-specification#347](https://github.com/open-telemetry/opentelemetry-specification/pull/347).
+3. Drop all label keys, and aggregate all measurements from the metric instrument together regardless of their labelsets.
```

Comment:
To clarify, this means that the exported metrics will have no labels associated with them correct?